packages/font/src/google/loader.ts

```diff
@@ -76,9 +76,7 @@ const downloadGoogleFonts: FontLoader = async ({
   if (adjustFontFallback) {
     try {
       const { ascent, descent, lineGap, fallbackFont, sizeAdjust } =
-        calculateSizeAdjustValues(
-          require('next/dist/server/google-font-metrics.json')[fontFamily]
-        )
+        calculateSizeAdjustValues(fontFamily)
```

Comment:
Just realized this may cause unexpected behaviour if updating just next or @next/font. I need to figure out how to make a nice error for this before merging I think.